gateleen-routing/src/main/java/org/swisspush/gateleen/routing/Router.java

```diff
@@ -85,6 +86,8 @@ public class Router implements Refreshable, LoggableResource, ConfigurationResou
     private OAuthStrategy oAuthStrategy = null;
     private BasicAuthStrategy basicAuthStrategy;
 
+    private static GateleenExceptionFactory gateleenExceptionFactory = GateleenExceptionFactory.newGateleenThriftyExceptionFactory();
```

Comment:
I strongly advise to use constructor injection. We Must be able to hand in our custom implementation from application side.

Using it hardcoded, defeats the whole purpose of it.

Reply:
I agree on this comment